### PR TITLE
Treat emptystring userquota as valid value

### PR DESCRIPTION
**Pull request by butonic**
fixes https://central.owncloud.org/t/invalid-quota-for-ldap-user/12147/6

### CODE REVIEW

File: lib/User/UserEntry.php [hunk 1]
```diff
@@ -211,7 +211,10 @@ public function getQuota() {
 			\OC::$server->getLogger()->debug("No LDAP quota attribute configured", ['app' => 'user_ldap']);
 		} else {
 			$quota = $this->getAttributeValue ($attr);
-			if (!$this->verifyQuotaValue($quota)) {
+			if ($quota === '') {
+				\OC::$server->getLogger()->debug("No quota set for <{$this->getOwnCloudUID()}>", ['app' => 'user_ldap']);
+				$quota = null;
+			} else if (!$this->verifyQuotaValue($quota)) {
```
Comment (jvillafanez) [inline lib/User/UserEntry.php]: I'd prefer:
```
} else if ($quota !== null && !$this->verifyQuotaValue($quota)) {
```

Comment (fbassas) [reply lib/User/UserEntry.php]: essages disapear only with @jvillafanez suggestion:

} else if ($quota !== null && !$this->verifyQuotaValue($quota)) {

without first part of "if" messages still apear.

**Comment by mmattel** [comment]
Any update on this one?

**Comment by fbassas** [comment]
This has not been solved in10.0.7

**Comment by mmattel** [comment]
This could be closed in favour of #237
@butonic ?

**Comment by PVince81** [comment]
@butonic please check comments and clarify next steps so we can take over

**Comment by butonic** [comment]
@mmattel is correct. fixed by https://github.com/owncloud/user_ldap/pull/237

**Pull request closed without merging**